File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/main/java/simulacionBandaMusica/simulacionBandaMusica.java
package simulacionBandaMusica;

import simulacionBandaMusica.factories.BandaFactory;
import simulacionBandaMusica.models.Banda;

import static simulacionBandaMusica.utils.funcionesMenu.menuBandaMusica;

public class simulacionBandaMusica {
    public static void main(String[] args) {
        // Instanciamos la banda
        Banda banda = BandaFactory.create();
        menuBandaMusica(banda.vectorMusicos());
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/main/kotlin/simulacionBandaMusica/simulacionBandaMusica.kt
package simulacionBandaMusica

import simulacionBandaMusica.factories.BandaFactory
import simulacionBandaMusica.utils.funcionesMenu

fun main(args: Array<String>) {
    // Instanciamos la banda
    val banda = BandaFactory.create()
    funcionesMenu.menuBandaMusica(banda.vectorMusicos)
}
